src/main/java/com/google/jenkins/plugins/googlecontainerregistryauth/GoogleContainerRegistryCredentialModule.java

```diff
@@ -78,7 +78,7 @@ public Secret getToken(GoogleRobotCredentials credentials) {
    * @return whether the credential could be applied to the given requirements.
    */
   public static boolean matches(List<DomainRequirement> requirements) {
-    Jenkins jenkins = Jenkins.getInstance();
+    Jenkins jenkins = Jenkins.get();
     if (jenkins == null) {
```

Comment:
This check should not be required anymore. `Jenkins.get()` does the instance check by default